benchmarks/domains/synthetic_2C1D_1C.py

```diff
@@ -60,24 +62,26 @@ def synthetic_2C1D_1C(settings: ConvergenceExperimentSettings) -> DataFrame:
         NumericalDiscreteParameter("z", (1, 2, 3, 4)),
     ]
 
-    objective = NumericalTarget(name="target", mode=TargetMode.MAX).to_objective()
-    search_space = SearchSpace.from_product(parameters=parameters)
+    target = NumericalTarget(name="target", mode="MAX")
+    searchspace = SearchSpace.from_product(parameters=parameters)
 
     scenarios: dict[str, Campaign] = {
         "Random Recommender": Campaign(
-            searchspace=search_space,
+            searchspace=searchspace,
             recommender=RandomRecommender(),
-            objective=objective,
+            objective=target,
         ),
         "Default Recommender": Campaign(
-            searchspace=search_space,
-            objective=objective,
+            searchspace=searchspace,
+            objective=target,
         ),
     }
 
+    lookup = label_columns([p.name for p in parameters], [target.name])(_lookup)
```

Comment:
it doestn matter, but wouldnt you prefer using it as a decorator above? In any case here I would actually prefer if the function was NOT coded in array style but already in df style... any reason why not?

Reply:
Please see my question [here](https://github.com/emdgroup/baybe/pull/441#discussion_r1871120433). I'm open for everything, even dropping the utility altogether. My primary question is: how do we generally want to construct these lookups in our code and examples?

I'll keep this thread open as a reminder but please answer in the other one

Reply:
imo its obvious that lookups should follow the new design, except where we already have something prepared like the botorch functions where the wrapper would indeed be useful

Reply:
See my answers in the other threads: Should have the stuff in general DataFrame based, using the decorator where it makes sense. Here, I'd prefer to keep the decorator as this reduces the amount of re-writing and the benchmarking is anyhow "semi-internal"